util/caller/resolver.go

```diff
@@ -45,8 +45,17 @@ type CallResolver struct {
 }
 
 // defaultPattern strips src/github.com/organization/project/module/submodule/file.go
-// to module/submodule/file.go.
-var defaultPattern = "src/" + strings.Repeat(strings.Join([]string{"[^", "]+", ""}, string(os.PathSeparator)), 3) + "(.*)"
+// down to module/submodule/file.go.
+var defaultPattern = func() string {
+	_, file, _, ok := runtime.Caller(0)
+	if !ok {
+		panic("unable to look up location")
+	}
+	const sep = string(os.PathSeparator)
+	path := filepath.Dir(file)
+	path = filepath.Join(path, strings.Repeat("../", 5)) + sep // $GOPATH/src/
```

Comment:
This `"../"` looks non-portable to me. I'm not sure yet what the portable equivalent is. Looking.